docs/topics/rest-level-0-mission-modify-wf-swarm.adoc

```diff
@@ -1,15 +1,15 @@
-= Modify the WildFly Swarm Quickstart to Add Additional HTTP Endpoints
+= Modify the {WildFlySwarm} Booster to Add Additional HTTP Endpoints
 
 
-include::rest-level-0-experience-modify-usecase.adoc[]
+include::rest-level-0-mission-modify-usecase.adoc[]
 
-To modify the WildFly Swarm quickstart:
+To modify the {WildFlySwarm} booster:
 
-. Add a `WorldGreeting` class to `src/main/java/org/obsidiantoaster/quickstart/rest/`.
+. Add a `WorldGreeting` class to `src/main/java/org/obsidiantoaster/booster/rest/`.
 +
 [source,java,options="nowrap",subs="attributes+"]
 ----
-package org.obsidiantoaster.quickstart.rest;
+package org.obsidiantoaster.booster.rest;
```

Comment:
Package should be `io.openshift.booster.rest`. Better avoid `org.obsidiantoaster` in the docs